src/custom/hooks/usePriceImpact/calculatePriceImpact.test.ts

```diff
@@ -0,0 +1,53 @@
+import { ChainId, WETH } from '@uniswap/sdk'
+import { CurrencyAmount, Percent, Token } from '@uniswap/sdk-core'
+import BigNumber from 'bignumber.js'
+import { parseUnits } from 'ethers/lib/utils'
+
+function _calculateAbaPriceImpact(initialValue: string, finalValue: string) {
```

Comment:
I feel a bit gilty you are now using ABA here and there as a name. This concept doen't exist, and I'm not sure what's the best way to call it. Usually not a fan of adding acronyms without context

If there's no way to give a better name and make it short, then I would add some minimal comment with info on what it is, and maybe a link to the issue where is defined.

Reply:
sure thing